test/model.indexes.test.js

```diff
@@ -70,6 +70,15 @@ describe('model', function() {
       });
     });
 
+    it('should result in a clean error when calling diffIndexes() against a collection that does not exist', async () => {
```

Comment:
Mongoose 5.x supports back to Node 4, so you can't use async functions in tests.